examples/cxx/publish_subscribe_with_user_header/src/publisher.cpp

```diff
@@ -39,10 +39,10 @@ auto main() -> int {
         counter += 1;
         auto sample = publisher.loan_uninit().expect("acquire sample");
 
-        sample.user_header_mut().version = 123;               // NOLINT
-        sample.user_header_mut().timestamp = 80337 + counter; // NOLINT
+        sample.user_header_mut().version = 123;                                      // NOLINT
+        sample.user_header_mut().timestamp = static_cast<uint64_t>(80337 + counter); // NOLINT
```

Comment:
I think in this case the type of the `counter` could be set to `uint64_t`, then the casts could become superfluous